cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -691,3 +691,80 @@ subjects:
   namespace: rook-ceph
 # OLM: END CMD REPORTER ROLEBINDING
 ---
+#
+# OLM: BEGIN CLUSTER ROLE
+---
+# Aspects for creation of monitoring resources
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: rook-ceph-monitor
+rules:
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - '*'
+  verbs:
+  - '*'
+---
+# OLM: END CLUSTER ROLE
+#
+# OLM: BEGIN CLUSTERROLE BINDING
+---
+# Allow creation of monitoring resources
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: rook-ceph-monitor
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: rook-ceph-monitor
+subjects:
+- kind: ServiceAccount
+  name: rook-ceph-system
+  namespace: rook-ceph
+---
+# OLM: END CLUSTERROLE BINDING
+#
+# OLM: BEGIN CLUSTER ROLE
+---
+# Aspects for metrics collection
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: rook-ceph-metrics
```

Comment:
This role is to allow Prometheus to scrape the metrics? How are the metrics working today if we haven't been granting this to the Prometheus SA?

Reply:
It doesn't work. Prometheus would have to be in the same namespace as that of the rook cluster (not always the case).

Reply:
I agree that a separate service account is good for users who don't want this functionality, and along those lines would it be better to put these RBAC configs into the manifest which enables prometheus monitoring.

Reply:
Agreed.